web/web.ex

```diff
@@ -38,6 +38,7 @@ defmodule CodeCorps.Web do
       import CodeCorps.Gettext
 
       import Canary.Plugs
+      import CodeCorps.AuthenticationHelpers, only: [load_and_authorize_changeset: 2]
```

Comment:
I was actually thinking of just extending `Canary.Plugs` and importing that module alone. Not sure what the best approach would be.